packages/apps/human-app/server/src/modules/oracle-discovery/oracle-discovery.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Logger,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { OracleDiscoveryService } from './oracle-discovery.serivce';
+import {
+  OracleDiscoveryCommand,
+  OracleDiscoveryResponse,
+  OracleDiscoveryDto,
+} from './interface/oracle-discovery.interface';
+
+@Controller()
+export class OracleDiscoveryController {
+  logger = new Logger(OracleDiscoveryController.name);
+  constructor(
+    private readonly service: OracleDiscoveryService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+  @ApiTags('Oracle-Discovery')
+  @Get('/oracles')
+  @ApiOperation({ summary: 'Operator signup' })
+  @UsePipes(new ValidationPipe())
+  public oracleDiscovery(
+    @Body() oracleDiscoveryDto: OracleDiscoveryDto,
+  ): Promise<OracleDiscoveryResponse[]> {
+    this.logger.log(
```

Comment:
if we need this logger, let it stay, but perhaps it would be better to create a new task to add logging